packages/opentelemetry-instrumentation-openai/opentelemetry/instrumentation/openai/shared/chat_wrappers.py

```diff
@@ -72,6 +72,7 @@ def chat_wrapper(
         kind=SpanKind.CLIENT,
         attributes={SpanAttributes.LLM_REQUEST_TYPE: LLM_REQUEST_TYPE.value},
     )
+    context_api.attach(trace.set_span_in_context(span))
```

Comment:
@nirga I saw this in several places in your latest PR - was it accidentally deleted at some point, or it should not be added? Please confirm.

Reply:
Lot of tests were failing, so I removed it.